kart/point_cloud/import_.py

```diff
@@ -124,8 +137,39 @@ def point_cloud_import(ctx, ds_path, sources):
             "crs84_envelope": crs84_envelope,
         }
 
+        if schema is None:
+            crs_name = get_identifier_str(crs_set[0])
+            schema = metadata["filters.info"]["schema"]
+            schema["CRS"] = crs_name
+
     click.echo()
 
+    version = version_set[0]
```

Comment:
and, for these ones where they _don't_ store dicts, maybe just use a normal `set()` ?

Reply:
I could but there's no real upside - it doesn't even make the code simpler